packages/iota/src/validation/block.ts

```diff
@@ -12,6 +13,12 @@ import type { IValidationResult } from "./result";
  * @returns The validation result.
  */
 export function validateBlock(block: IBlock, protocolInfo: INodeInfoProtocol): IValidationResult {
-    return validatePayload(block.payload, protocolInfo);
+    let payloadResult: IValidationResult = { isValid: true };
+    let parentsResult: IValidationResult = { isValid: true };
```

Comment:
I see no need for `let` then assign here, just take the result and assign it inline with `const`.